docs/appendix/changelog.md

```diff
@@ -19,24 +19,58 @@ outputs:
 
 <!-- If there is no concrete date for a change that makes sense, use the end of the month it was released in. -->
 
-<!-- This isn't in the docs yet
-{{% changelog date="2024-06-14" color="changed" title="Python get_robot_part_logs parameters" %}}
+{{% changelog date="2024-09-06" color="added" title="Sample datasets" %}}
 
-The `errors_only` parameter has been removed from [`get_robot_part_logs()`](/appendix/apis/fleet/#getrobotpartlogs) and replaced with `log_levels`.
+You can now get a sample dataset from the docs.
+For more information, see [Sample dataset](/services/data/dataset/#sample-dataset).
 
-{{% /changelog %}}-->
+{{% /changelog %}}
 
-<!-- This isn't in the docs yet
-{{% changelog date="2024-05-28" color="added" title="CaptureAllFromCamera and GetProperties to vision API" %}}
+{{% changelog date="2024-08-26" color="added" title="ESP32 cameras" %}}
 
-The vision service now supports two new methods: [`CaptureAllFromCamera`](/services/vision/#captureallfromcamera) and [`GetProperties`](/services/vision/#getproperties).
+`viam-micro-server` now supports cameras on ESP32s.
+For more information, see [Configure an esp32-camera](/components/camera/esp32-camera/).
+
+{{% /changelog %}}
+
+{{% changelog date="2024-08-26" color="changed" title="Micro-RDK now called viam-micro-server" %}}
+
+The micro-RDK is now referred to as `viam-micro-server`.
+For more information, see [viam-micro-server](/architecture/viam-micro-server/).
+
+{{% /changelog %}}
+
+{{% changelog date="2024-08-26" color="added" title="Provisioning" %}}
+
+You can now configure provisioning for machines with the Viam Agent.
+For more information, see [Configure provisioning with viam-agent](/how-tos/provision-setup/).
+
+{{% /changelog %}}
+
+{{% changelog date="2024-08-16" color="added" title="Data capture for vision" %}}
+
+Data capture is now possible for the vision service.
+For more information, see [Supported components and services](/services/data/capture/#supported-components-and-services).
 
-{{% /changelog %}}-->
+{{% /changelog %}}
+
+{{% changelog date="2024-08-01" color="added" title="Create custom training scripts" %}}
+
+You can now create, upload and submit custom training scripts to the Viam Registry.
```

Comment:
That doesn't make sense, you submit jobs not trainign scripts
```suggestion
You can now upload custom training scripts to the Viam Registry and use them to train machine learning models.
```

Reply:
thanks